ethcore/src/miner/transaction_queue.rs

```diff
@@ -557,7 +571,7 @@ pub struct TransactionQueue {
 	/// The maximum amount of gas any individual transaction may use.
 	tx_gas_limit: U256,
 	/// Current gas limit (block gas limit * factor). Transactions above the limit will not be accepted (default to !0)
-	gas_limit: U256,
+	block_gas_limit: U256,
```

Comment:
Why the rename? This is still a queue gas limit, not the block gas limit. `total_gas_limit` might be a better alternative.